preprocess.py

```diff
@@ -1,16 +1,32 @@
+import warnings
+# Ignore future warnings by librosa in resemblyzer
+from collections import Counter
+from typing import Tuple
+
+from utils.text.recipes import read_ljspeech_format, read_metadata
+
+warnings.simplefilter(action='ignore', category=FutureWarning)
+
 import argparse
+import traceback
 from dataclasses import dataclass
 from multiprocessing import Pool, cpu_count
 from random import Random
 
-import pyworld as pw
+import tqdm
+import torch
+from resemblyzer import VoiceEncoder
+from resemblyzer import preprocess_wav as preprocess_resemblyzer
 
+from pitch_extraction.pitch_extractor import PitchExtractor, new_pitch_extractor_from_config
 from utils.display import *
 from utils.dsp import *
 from utils.files import get_files, pickle_binary, read_config
 from utils.paths import Paths
 from utils.text.cleaners import Cleaner
-from utils.text.recipes import ljspeech
+
+
+SPEAKER_EMB_DIM = 256
```

Comment:
Can we read this from the config?

Reply:
Could be done but the dim is fixed due to the external dependency of resemblyzer. Added a comment.